src/cli/project/environment/add.rs

```diff
@@ -26,7 +27,7 @@ pub struct Args {
 
 pub async fn execute(mut project: Project, args: Args) -> miette::Result<()> {
     let environment_exists = project
-        .environment(&EnvironmentName::Named(args.name.clone()))
+        .environment(&EnvironmentName::from_str(args.name.as_str())?)
```

Comment:
Could you change the type of `args.name` to `EnvironmentName` like we do with `pixi global`?

Reply:
Applied my suggestion in af2bc1f286a831e61881b9f2bb49ebd8b0fca682